framework/include/utils/RankFourTensorImplementation.h

```diff
@@ -833,17 +841,21 @@ RankFourTensorTempl<T>::fillAxisymmetricRZFromInputVector(const std::vector<T> &
   mooseAssert(input.size() == 5,
               "To use fillAxisymmetricRZFromInputVector, your input must have size 5.");
 
-  // C1111     C1122     C1133     C2222     C2233=C1133
-  fillSymmetric9FromInputVector({input[0],
-                                 input[1],
-                                 input[2],
-                                 input[0],
-                                 input[2],
-                                 // C3333     C2323     C3131=C2323   C1212
-                                 input[3],
-                                 input[4],
-                                 input[4],
-                                 (input[0] - input[1]) * 0.5});
+  // C1111  C1122     C1133       0         0         0
+  //        C2222  C2233=C1133    0         0         0
+  //                  C3333       0         0         0
+  //                            C2323       0         0
+  //                                   C3131=C2323    0
+  //                                                C1212
+  // clang-format off
+  fillSymmetric21FromInputVector(std::array<T, 21>
+  {{input[0],input[1],input[2],      0.0,      0.0, 0.0,
+             input[0],input[2],      0.0,      0.0, 0.0,
+                      input[3],      0.0,      0.0, 0.0,
```

Comment:
I'll lean on @dschwen here...but is it possible to make this a smaller array by using references for duplicative values? Maybe that is what is going on already...just thinking out loud.

Reply:
I don't think so.

Reply:
It was calling fillSymmetric9fromInputVector before which was constructing a smaller array but I thought fill symmetric 21 would be better since it doesn't have to call a zero on the rank four tensor.  I also think the upper triangular 6x6 format look nice.